pymongo/srv_resolver.py

```diff
@@ -83,8 +85,8 @@ def get_options(self):
 
     def _resolve_uri(self, encapsulate_errors):
         try:
-            results = _resolve('_mongodb._tcp.' + self.__fqdn, 'SRV',
-                               lifetime=self.__connect_timeout)
+            results = _resolve('_' + self.__srv + '._tcp.' + self.__fqdn,
+                                'SRV', lifetime=self.__connect_timeout)
```

Comment:
The indentation here still looks off, should be:
```python
            results = _resolve('_' + self.__srv + '._tcp.' + self.__fqdn,
                               'SRV', lifetime=self.__connect_timeout)
```

Reply:
Done.